### PR TITLE
bugfix-84-movement-bar

### PR DESCRIPTION
**Pull request by Kseniia-Koshkina**

### CODE REVIEW

**Review by DZotoff — APPROVED** [review]
Accepted! Good job!

File: src/components/timebank/timebank-content.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ const TimebankContent = ({ handleDailyEntryChange, loading }: Props) => {
     return (
       <>
         <TimebankOverviewChart personTotalTime={personTotalTime} />
-        <List dense sx={{ marginLeft: "5%" }}>
+        <List style={{width:"12%", minWidth:"110px"}} dense sx={{ marginLeft: "5%" }}>
```
Comment (Jdallos) [inline src/components/timebank/timebank-content.tsx:131]:
```suggestion
        <List style={{ width:"12%", minWidth:"110px" }} dense sx={{ marginLeft: "5%" }}>
```

File: src/components/timebank/timebank-content.tsx [hunk 2]
```diff
@@ -367,10 +367,10 @@ const TimebankContent = ({ handleDailyEntryChange, loading }: Props) => {
             <TimebankCardFlexBox>
               {renderDatePickers()}
               <FormControlLabel
-                sx={{ display: "inline" }}
+                style={{ width:"20%" }}
                 label={strings.timebank.byrange}
                 control={
-                  <Checkbox
+                  <Checkbox
```
Comment (Jdallos) [inline src/components/timebank/timebank-content.tsx:373]:
```suggestion
                  <Checkbox                    
```

**Review by Jdallos — APPROVED** [review]
Approved but conflicts need resolving before this can be merged

**Pull request merged**